From 65e1f05b9636d85f390915e283ed8cf69dd6ea09 Mon Sep 17 00:00:00 2001
From: Yann Dirson <ydirson@free.fr>
Date: Sun, 6 Aug 2017 18:05:26 +0200
Subject: [PATCH] Honor standard CPPFLAGS

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 485ee61..666e258 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ ALL= hachu hachu.6.gz
 all: ${ALL}
 
 hachu: hachu.c
-	$(CC) $(CFLAGS) $(LDFLAGS) hachu.c -o hachu
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) hachu.c -o hachu
 
 install: ${ALL} ${srcdir}/svg/*
 	install -d -m0755 $(DESTDIR)$(prefix)/games
-- 
2.13.2

